### ISSUE TITLE
I Found A Bug On Elementor's Free Plugin. It's About Responsiveness From 993px till 1199px.

### ISSUE THREAD

**Issue by DaljitKhalsa** [elementor/elementor#17114]
**I FOUND A BUG ON** Elementor **WHICH EVERYONE OF US MIGHT** be having. 

Here is the video I just created about this bug = **_https://youtu.be/MeASeoAfAnw_**

Note: Please watch the video till the end, to see that I have tried everything.

**If you can not watch the video at work computer then my issue is written below.**

How to force Elementor to break the header to hamburger below 1200px till 991px? 

In other words, from 0px till 992px the hamburger shows properly on all the Elementor sites but after 993px till 1199px most of us have issues with the header area.
